### ISSUE TITLE
Should packages be built locally before documenting?

### ISSUE THREAD

**Issue by rkent** [ros-infrastructure/rosdoc2#80]
The current rosdoc32 instructions say:
```
 To generate the documentation for almost any ROS 2 package, first build the package locally, ...
```
Is it necessary or useful to build the document locally first? Does the build farm do that? If not, we should change the instructions.

**Comment by Yadunund** [comment]
We do not need to build the package locally. The buildfarm does not do that either so we should update the text in the README accordingly.

**Comment by tfoote** [comment]
Yeah, that's outdated. I think we used to build first so that we would generate the API docs for the generated message types, but we're not doing that now. We could consider trying to run the message generation to fill that out. But we don't want to have to build everything before documenting.

**Comment by tfoote** [comment]
Fixed in #86
